pkg/trace/api/pipeline_stats.go

```diff
@@ -126,6 +136,11 @@ func (m *multiDataStreamsTransport) RoundTrip(req *http.Request) (rresp *http.Re
 			// given the way we construct the list of targets the main endpoint
 			// will be the first one called, we return its response and error
 			rresp, rerr = m.rt.RoundTrip(newreq)
+			if rerr != nil {
+				log.Errorf("[pipeline_stats] RoundTrip failed: %v", rerr)
```

Comment:
do you need to be able to differentiate between errors at line 120 and line 140, or is this sufficient?

Reply:
thanks for taking a look! I think the two errors should be the same because the roundtrip call on line 118 looks to be a "shortcut" case for when there's only 1 target.